ParisBNF/abb/BNFabb114.xml

```diff
@@ -58,6 +58,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         </extent>
                         <foliation>There is foliation provided in the upper right corner of each folio.</foliation>
                      </supportDesc>
+                     <layoutDesc>  
+                        <layout columns="2" writtenLines="20">
+                           <locus from="8ra" to="108va"/>
+                           <ab type="pricking">Pricking and ruling are visible.</ab>
+                           <ab type="pricking">Primary pricks are visible, horizontal pricks are visible.</ab>
```

Comment:
You need to include the formula for the ruling.

Reply:
Okay thank you, I'll.